### ISSUE TITLE
Add information about deploy process

### ISSUE THREAD

**Issue by grelement** [wildland/guides#13]
[] setting up new heroku instance
[] pushing changes to heroku

**Comment by jweakley** [comment]
Saucelab testing: https://github.com/wildland/distilltech-web/pull/62

**Comment by jweakley** [comment]
Docker containers - http://blog.carbonfive.com/2015/03/17/docker-rails-docker-compose-together-in-your-development-workflow/#user-content-adding-the-app
TL;DR
- Download and install [VirtualBox](https://www.virtualbox.org/wiki/Downloads).
- `brew install boot2docker docker-compose`
- `boot2docker init`
Note that there is a bug that is causing this to not work: https://github.com/boot2docker/boot2docker/issues/982
- `boot2docker start`
- `$(boot2docker shellinit)` Add to .bashrc?

Add `Dockerfile` to project.

**Comment by jweakley** [comment]
Sauce lab testing: https://github.com/johanneswuerbach/ember-cli-sauce
TL;DR
- Set ENV vars: `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY`
- Have tunnel open with `ember start-sauce-connect`
- Run `ember test`
- Run `ember stop-sauce-connect` to close the tunnel after testing.

Defining a browser: `ember sauce --help
`. Example for IE on windows 8.1: `ember sauce -b 'internet explorer' -p 'Windows 8.1'`.
- Add this defined browser to the CI list in `testem.json`
